src/migration/utilities.js

```diff
@@ -1,14 +1,32 @@
-function getDependency(artifacts, network, deployer, path) {
+function initializeContract(path, deployer, accounts) {
+  const contract = require("truffle-contract")
+
+  const Contract = contract(require(path))
```

Comment:
Is there a reason we name instance with uppercase and the imported libraries wit lower-case? I was under the assumption we usually do it the other way around.